data/faq/application_template.md

```diff
@@ -0,0 +1,108 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
+- Time Zone:
+- GSoC blog RSS feed URL:
+    (You will have to write a blog post every week during GSoC.)
+
+
+Code Sample
+-----------
+
+Link all your contributions to coala and any other open source project here.
+For example, to list your contributions to coala you can mention:
+
+- https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
+for coala repo
+- https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
+for coala-bears repo
+
+You may also mention specific issues/commits using the commit id if you want
+us to take a look at that.
+
+You will have to finish the coala newcomer guide to be accepted:
+http://coala.io/newcomer
+
+
+Project Info
+------------
+
+Which project from http://projects.coala.io are you applying for?
+If you have your own idea, please add it to our projects list:
+https://github.com/coala/projects/blob/master/data/projects.js
+
+If you are afraid of someone else stealing your awesome idea, don't be. We
+can see who proposed the idea.
+Add all ideas you have to solve the project here. This could include
+research on linters, libraries and similar problems as well as mockups
+of code, api and ui changes.
+
+Especially: Provide milestones for your project. Ideally weekly steps. This
```

Comment:
why not have the milestones as a subheading?

I think we should ask a bit more about the project, take a look at what GNOME does: https://wiki.gnome.org/Outreach/SummerOfCode/Students

- asking people about if they already engaged with their mentor, which mentor do they wish
- what's the ultimate goal
- how is their project helping our users
- what stuff will they need to touch

all those things that show they have thought about this beyond just copying the description